extensions/notebook/package.nls.json

```diff
@@ -33,8 +33,8 @@
 	"title.saveJupyterBook": "Save Book",
 	"title.trustBook": "Trust Book",
 	"title.searchJupyterBook": "Search Book",
-	"title.SavedBooks": "Saved Books",
-	"title.UnsavedBooks": "Unsaved Books",
+	"title.SavedBooks": "Notebooks",
+	"title.UnsavedBooks": "Provided Books",
```

Comment:
Update these keys as well - let's not leave stale references to "unsaved" books around.

Did you also do a scrub of the code to update any comment/variable references? I see at least a couple such as the context key that should also be updated

Reply:
I'll scrub them in a different PR, wanted to reduce the amount of code that's getting merged to release at this point.